metabase/metabase
# Upgrade webpack-dev-server

Depending on the code you change, the dev server crashes periodically. Increasing NodeJS memory limit does not help.
Let's try the newest version of the `webpack-dev-server` and see if it helps.

### How to verify
- Run webpack in the dev mode
- Ensure everything works as expected
